src/js/utils.js

```diff
@@ -20,7 +44,8 @@ this.exports = {
             {
                 "type": "setHold",
                 "params": {
-                  "holdType": "indefinite", 
+                  "holdType": this.calculateHoldType(thermostat.settings), 
```

Comment:
So I tested with not sending a the hold type or hours and it defaults to indefinite so I think we should keep this in.